packages/generator-one-app-module/generators/app/index.js

```diff
@@ -63,8 +69,40 @@ module.exports = class extends Generator {
       type: 'input',
       name: 'moduleName',
       validate: helper.validateIfInputIsValidOrNot,
-      message: 'What is the name of your Module?',
-    }];
+      message: 'Name of your Module:',
+      store: false,
+    },
+    {
+      type: 'list',
+      name: 'moduleType',
+      default: 'root module',
+      message:
+        'Is this a root module or child module?',
+      choices: ['root module', 'child module'],
+      store: false,
+    },
+    {
+      type: 'list',
+      name: 'setupParrotMiddleware',
+      default: 'Yes',
+      message:
+        'Generate with Parrot Middleware?',
```

Comment:
Can we add a link to what a parrot middleware is?

Reply:
https://github.com/americanexpress/one-app-cli/pull/25#discussion_r382259026